packages/web-client/README.md

```diff
@@ -0,0 +1,3 @@
+# web-client
+
+This is experimental.
```

Comment:
I will follow-up on this in another PR, together with dev docs about how to use the package.